testdata/source/schwab_csv/test_basic/import_results.beancount

```diff
@@ -1,23 +1,13 @@
 ;; date: 2018-06-28
 ;; info: {"filename": "<testdata>/test_basic/transactions/Intelligent_Transactions_20201115-180120.CSV", "line": 17, "type": "text/csv"}
 
-; features: [
-;             {
-;               "amount": "-0.05 USD",
-;               "date": "2018-06-28",
-;               "key_value_pairs": {
-;                 "desc": "INTEREST 05/30THRU 06/27",
-;                 "schwab_action": "Margin Interest"
-;               },
-;               "source_account": "Assets:Schwab:Intelligent-4321:Cash"
-;             }
-;           ]
-2018-06-28 * "MARGININTEREST - INTEREST 05/30THRU 06/27"
+; features: []
+2018-06-28 * "Margin Interest - INTEREST 05/30THRU 06/27"
   Assets:Schwab:Intelligent-4321:Cash  -0.05 USD
     date: 2018-06-28
     schwab_action: "Margin Interest"
     source_desc: "INTEREST 05/30THRU 06/27"
-  Expenses:FIXME                        0.05 USD
+  Income:Interest:Schwab                0.05 USD
```

Comment:
I can see that the right account is filled in here, but it's strange that the features array is now empty. Is this a regression?